content/citation-tags.tsv

```diff
@@ -80,6 +80,7 @@ Gerstein2016_scaling	doi:10.1186/s13059-016-0917-0
 Ghandi2014_enhanced	doi:10.1371/journal.pcbi.1003711
 Ghosh1992_sequence	doi:10.1117/12.140112
 Glorot2011_domain	url:http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.231.3442
+Goh2017_compchemistry	doi:10.1002/jcc.24764
```

Comment:
We can remove the tag if the DOI is being using in the reference above. The tag is only needed when we use it directly in the citation from Markdown.